config/serverless.security.search_ai_lake.yml

```diff
@@ -18,6 +37,38 @@ xpack.features.overrides:
   siemV2.description: null
   securitySolutionSiemMigrations.hidden: true
 
+## Fine-tune the security solution essentials feature privileges. These feature privilege overrides are set individually for each project type. Also, refer to `serverless.yml` for the project-agnostic overrides.
+  dashboard:
+    privileges:
+      ## Setting to empty array so the values from serverless.yml or serverless.security.yml are overwritten
+      all.composedOf: []
+      read.composedOf: []
+  dashboard_v2:
+    privileges:
+      ## Setting to empty array so the values from serverless.yml or serverless.security.yml are overwritten
+      all.composedOf: []
+      read.composedOf: []
+  siemV2:
+    privileges:
+      all.composedOf:
```

Comment:
>[!WARNING]
> In other tiers SIEM and SIEM_V2 also automatically grant `dashboard_v2` in addition to `discover_v2`, is it not the case for this tier?

Reply:
You're right. Dashboard will be disabled in the follow up PR. 
Bringing it back.